Project-Code/src/index.js

```diff
@@ -59,26 +58,36 @@ const user = {
 
 
 app.get('/', (req, res) =>{
-    
     res.render('pages/login')
 });
 
 app.get('/test', (req, res)=>{
```

Comment:
It's common to have a folder that has "controllers", and you can pass each "controller as an argument to `app.get`,  so you can then use smaller functions.

just pass the data in the response to each subsequent caller.

Reply:
This can also be its own folder like `/controllers`